python-package/lightgbm/dask.py

```diff
@@ -510,7 +570,7 @@ def fit(self, X, y, sample_weight=None, client=None, **kwargs):
                    + ' ' * 12 + 'Dask client.\n'
                    + ' ' * 8 + _init_score + _after_init_score)
 
-    def predict(self, X, **kwargs):
+    def predict(self, X: _DaskCollection, **kwargs) -> _DaskCollection:
```

Comment:
Based on https://github.com/microsoft/LightGBM/pull/3866/files#r565691112.
```suggestion
    def predict(self, X: _DaskCollection, **kwargs) -> dask_Array:
```